PennMobile/Setup + Navigation/PennMobile.swift

```diff
@@ -53,6 +54,9 @@ struct PennMobile: App {
                 .environmentObject(mockHomeViewModel)
             #endif
                 .accentColor(Color("navigation"))
+            
+            ContentView()
+                .environmentObject(popupManager)
```

Comment:
Let's move this into `RootView` - we can add an `.overlay` modifier like we do for toasts to keep the code cleaner here.